spec/integration/user_show_spec.rb

```diff
@@ -3,7 +3,7 @@
 RSpec.describe 'Test for the user show page', type: :system do
   context ' Check the right content for users show page' do
     before(:each) do
```

Comment:
- [ ] Your user show page test is missing the scenario.

  - Tests that redirects to the post's show page when you click on a user's post.